README.md

```diff
@@ -32,7 +32,7 @@ This project hosts each sample app in separate repository branches. For more inf
 
 ## Screenshots
 
-<img src="screenshots/todoapp.gif" alt="Screenshot">
```

Comment:
Can we safely remove the old screenshot now?

Reply:
I decided to leave it in because they're still currently indexed and served to pages, I'll do a follow up CL to remove them once the new screenshots have been indexed. A cron job runs daily for this, so conservative TTL is 36 hrs.